src/components/z-status-tag/styles.css

```diff
@@ -0,0 +1,132 @@
+z-status-tag {
+  --z-icon-width: 14px;
+  --z-icon-height: 14px;
+
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+
+  height: 22px;
+  border-radius: var(--border-radius);
+  padding-right: 4px;
+  padding-left: 4px;
+  text-transform: uppercase;
+
+  display: inline-flex;
+  align-items: center;
+}
+
+z-status-tag.expandable > z-icon {
+  margin-right: 0;
+  transition: margin-right 1s;
+}
+
+z-status-tag:not(.expandable) > z-icon:not(:last-child),
+z-status-tag.expandable:hover > z-icon:not(:last-child) {
+  margin-right: var(--space-unit);
+}
+
+z-status-tag.expandable > span {
+  max-width: 0;
+  transition: max-width 1s;
```

Comment:
the transition duration is different from the abstract design document

Reply:
risolto